Machine.Specifications.ReSharper.Provider.10/MspecTestElementMapper.cs

```diff
@@ -21,15 +20,35 @@ public class MspecTestElementMapper : DotNetCoreTestElementMapper
         public MspecTestElementMapper(
             IProject project,
             TargetFrameworkId targetFrameworkId, 
-            UnitTestElementFactory factory) 
-            : base(project, targetFrameworkId)
+            UnitTestElementFactory factory) : base(project, targetFrameworkId)
         {
             _project = project;
             _targetFrameworkId = targetFrameworkId;
             _factory = factory;
         }
 
-        protected override IUnitTestElement Map(Test test)
+        private static string GetType(Test test)
+        {
+            return test.FullyQualifiedName.SubstringBeforeLast("::").Trim();
+        }
+
+        private static string GetField(Test test)
```

Comment:
No need for these methods to be static.

Reply:
But it doesn't base on any property in a class so it would be better if it is a `private static`, also `GetType` is also marked as a `private static`